app/controllers/categories_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
   end
 
   def show
-    @category = Category.friendly.find(params[:category])
```

Comment:
Let's make sure we maintain functionality to keep the category name in the URL... So `/categories/scuba` instead of `/categories/3`. I think this will alter some other things on this PR so I'll hold off on making too many comments. I'll do a quick once over and see if anything else jumps out. Let me know when it's ready for another look.